gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/explore.proto

```diff
@@ -32,6 +32,7 @@ message ExploreRequest {
   int32 offset = 21;
 
   string space_id = 22;
+  bool include_non_live_entities = 23;
```

Comment:
Additional information for a given context, should be align as `oneOf` operator?
For now, it is`EntityOptions`. Later if we need to add for span or API_Trace, it will be `ApiTraceOptions`.

Also this is PR changes the API signature, so, it should be mark as `feat` or something else. Not as `chore`.